inst/stan/tv_normal_log.stan

```diff
@@ -1,47 +1,66 @@
+functions {
+  #include functions/prob_infected.stan
+}
+
 data {
-     int<lower=0> n_obs;
-     int n_pos[n_obs];
-     int n_total[n_obs];
-     int <lower=1>age_max;
-     matrix[n_obs, age_max] observation_exposure_matrix;
+  int<lower=0> n_obs;
+  int n_pos[n_obs];
+  int n_total[n_obs];
+  int <lower=1>age_max;
```

Comment:
for uniformity remove the space between int and <lower=1>
i.e., int<lower=1> age_max;